converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/FhirTransaction/ImagingStudy/IImagingStudyInstancePropertySynchronizer.cs

```diff
@@ -14,6 +15,7 @@ public interface IImagingStudyInstancePropertySynchronizer
         /// </summary>
         /// <param name="context">The transaction context.</param>
         /// <param name="instance">The instance component within study.</param>
-        void Synchronize(FhirTransactionContext context, ImagingStudy.InstanceComponent instance);
+        /// <param name="cancellationToken">The cancellation token.</param>
+        void Synchronize(FhirTransactionContext context, ImagingStudy.InstanceComponent instance, CancellationToken cancellationToken);
```

Comment:
looks like the operations are changed to async. in that case, 
1. should return Task instead of void
2. should name changes to xxxAsync

Reply:
Updated, thanks for the catch